.github/workflows/release-canary.yml

```diff
@@ -4,7 +4,7 @@ run-name: Canary release from ${{ github.ref_name }}
 on:
   push:
     branches:
-      - main
+      - release/v4
```

Comment:
🤔 We probably need to add both branched in main and add a condition for this action to block it in pre-release tags. Let's make this change in another PR.